application-development/helloworld-springboot-auth/pom.xml

```diff
@@ -0,0 +1,85 @@
+<!--
+~ Copyright (c) 2017 Google Inc. All Rights Reserved.
+~
+~ Licensed under the Apache License, Version 2.0 (the "License"); you
+~ may not use this file except in compliance with the License. You may
+~ obtain a copy of the License at
+~
+~     http://www.apache.org/licenses/LICENSE-2.0
+~
+~ Unless required by applicable law or agreed to in writing, software
+~ distributed under the License is distributed on an "AS IS" BASIS,
+~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+~ implied. See the License for the specific language governing
+~ permissions and limitations under the License.
+-->
+<project>
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.example.flex.gettingstarted</groupId>
+  <artifactId>helloworld-springboot</artifactId>
+  <version>0.0.1-SNAPSHOT</version>
+  <packaging>jar</packaging>
+
+  <name>helloworld-springboot</name>
+  <description>Demo project for Spring Boot</description>
+
+  <parent>
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-starter-parent</artifactId>
+    <version>1.5.5.RELEASE</version>
+  </parent>
+
+  <properties>
+    <java.version>1.8</java.version>
+    <maven.compiler.source>${java.version}</maven.compiler.source> <!-- REQUIRED -->
+    <maven.compiler.target>${java.version}</maven.compiler.target> <!-- REQUIRED -->
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <appengine.maven.plugin>1.3.1</appengine.maven.plugin>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-web</artifactId>
+      <version>1.4.4.RELEASE</version>
```

Comment:
the flex getting started example uses 1.5.8 (https://github.com/GoogleCloudPlatform/getting-started-java/blob/master/helloworld-springboot/pom.xml#L46)

Was I so late in reviewing this that they upgraded :)